Source/WebKit/Shared/Cocoa/WebCoreArgumentCodersCocoa.mm

```diff
@@ -497,6 +497,7 @@
     return nullptr;
 }
 
+// FIXME: This is duplicate code.
```

Comment:
@mattwoodrow just duplicated these like last week. Can we make sure he un-duplicates them soon?